pkg/metrics/resource.go

```diff
@@ -51,6 +53,17 @@ var (
 		Name: policyPreemptionMetricsName,
 		Help: "Number of preemption for the resource template. By the result, 'error' means a resource template failed to be preempted by other propagation policies. Otherwise 'success'.",
 	}, []string{"result"})
+	federatedHPADurationHistogram = prometheus.NewHistogramVec(prometheus.HistogramOpts{
+		Name:    federatedHPADurationMetricsName,
+		Help:    "Duration in seconds to process a FederatedHPA. By the result, 'error' means a FederatedHPA failed to be processed. Otherwise 'success'.",
+		Buckets: prometheus.ExponentialBuckets(0.001, 2, 12),
```

Comment:
This code snippet only records the time range from 0.001s to 4.096s(0.001* 2^12), which may not be sufficient for recording time cost as it could take longer than 4.096s. To address this, we can modify it to use `prometheus.ExponentialBuckets(0.01, 2, 12)`.

Reply:
OK